### ISSUE TITLE
Implement UI button control - use border box and fonts from rzxe

### ISSUE THREAD

**Issue by rozniak** [rozniak/Junkbot#4]
My plan is to use generalized user interface controls in Junkbot. The original game has an image per button used in the game, with the text already in place. Whilst it would be easy to just render these images, I think it would be nice to have the buttons be dynamically built, in case any new features get added to this version and we need custom button text.

This is kinda done right now... but it's not great - I implemented a rough `DrawBorderBox` method in rzxe that it makes use of right now, it was good for testing but it's less than ideal.

Essentially this is blocked by https://github.com/rozniak/rzxe/issues/1 - once the necessary feature is available in rzxe then this can be implemented (hopefully) quite easily.
